corgidrp/detector.py

```diff
@@ -27,6 +27,29 @@ def create_dark_calib(dark_dataset):
 
     return new_dark
 
+def dark_subtraction(input_dataset, dark_frame):
```

Comment:
I think this is maybe here because of an old merge, we've since moved this to l2a_to_l2b.py